frontends/PyCDE/integration_test/bsp/test_xrt.py

```diff
@@ -0,0 +1,42 @@
+# REQUIRES: xrt
+# RUN: rm -rf %t
+# RUN: %PYTHON% %s %t genhw 2>&1
+
+import pycde
+from pycde import (Clock, Input, module, generator, types)
+from pycde.bsp import XrtBSP
+
+import os
+import sys
+
+
+@module
+class Top:
+  clk = Clock(types.i1)
+  rst = Input(types.i1)
+
+  @generator
+  def construct(ports):
+    pass
+
+
```

Comment:
No input sanitization?

Reply:
Nah. It's just a test.

Reply:
How is this test run? Not pytest? Not part of a pipeline?
# RUN: %PYTHON% %s %t genhw 2>&1

This doesn't tell users what the arguments are and it's not hard to provide the help and checks on input arguments.

Reply:
Speaking as a former QA engineer, writing undocumented crappy tests is a good way to make sure those tests don't get ran over time and get deleted.

Reply:
It's a [`lit`](https://www.llvm.org/docs/CommandGuide/lit.html) test. Standard throughout LLVM/MLIR/CIRCT.